src/styles/blocks/header.scss

```diff
@@ -0,0 +1,54 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  text-transform: uppercase;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  &__item {
+    margin-right: 20px;
+  }
+
+  &__item:last-child {
+    margin-right: 0;
+  }
+
+  &__link {
+    position: relative;
+    text-decoration: none;
+    padding: 23px 0;
+    display: block;
+    font-weight: 500;
+    font-size: 12px;
+    color: black;
```

Comment:
```suggestion
    color: $main-accent-color;
```

Reply:
I know that in figma it was black, anyway according to the requirements and tests it should be like this :)